t/api/04-jobs.t

```diff
@@ -1222,6 +1225,39 @@ subtest 'show job modules execution time' => sub {
     is($testresults[0]->{execution_time},     '', 'the old format json file does not include execution_time');
 };
 
+subtest 'marking job as done' => sub {
```

Comment:
do we need another subtest? Don't we already have tests for setting something to done or incomplete jobs in this file?

Reply:
I couldn't find an explicit API test for it.

Reply:
fine then